app/services/keybase_service.rb

```diff
@@ -0,0 +1,27 @@
+class KeyBaseService
+
+  include AgentLogic
+
+  def initialize
+    @keybase_url = "https://keybase.io/"
+  end
+
+  def get_validator_avatar(validator)
+    image_css = '.picture > .img-circle'
+    keybase_url = @keybase_url + validator.keybase_id
+    keybase = get_page(url: keybase_url)
+
+    if keybase[:http_code] == '200'
+
+      if Nokogiri::HTML(keybase[:html]).css(image_css).present?
+        avatar_url = Nokogiri::HTML(keybase[:html]).css(image_css)
+                            .attribute('src').value
```

Comment:
If you're going to line break, line break all the method calls
```suggestion
        avatar_url = Nokogiri::HTML(keybase[:html])
                               .css(image_css)
                               .attribute('src').value
```